content/300-guides/200-deployment/110-deployment-guides/500-deploying-to-netlify.mdx

```diff
@@ -225,66 +291,77 @@ Admin URL: https://app.netlify.com/sites/YOUR_SITE_NAME
 URL:       https://YOUR_SITE_NAME.netlify.app
 ```
 
-> **Note:** You can change the build configuration via the `netlify.toml` file and the site name from the Netlify admin UI.
+> **Note:** You can change the build configuration in the `netlify.toml` file and the site name from the Netlify admin UI.
+
+Netlify deploys your app for the first time to the designated URL. Also, your application is now configured for continuous deployments. Every time you make a push to the GitHub repository, Netlify starts a new build of the site.
+
+You can call the API status endpoint with `curl` to veriy that your application is deployed.
+
+```terminal
+curl https://YOUR_SITE_NAME.netlify.app/.netlify/functions/status
+```
+
+The call should return: `{"up":true}`
+
+You can open your app now, but you will see errors when you use options that connect to the database. Netlify does not yet have the database connection string in a `DATABASE_URL` environment variable to establish a database connection.
+
+You can now set the `DATABASE_URL` environment variable in your Netlify site settings.
 
 ## 8. Expose the <inlinecode>DATABASE_URL</inlinecode> environment variable to functions
 
-Because Prisma will connect to the database from your serverless functions, you need to expose the `DATABASE_URL` to the backend functions. This is typically done from the Netlify UI.
+Because Prisma will connect to the database from your serverless functions, you need to expose the `DATABASE_URL` to the backend functions. You typically do this from the Netlify UI.
```

Comment:
I think one could also use the CLI to do that, so a future iteration that wants to be CLI only could use that instead: https://github.com/netlify/cli/blob/main/docs/commands/env.md